types/bunyan/index.d.ts

```diff
@@ -210,6 +210,9 @@ declare namespace Logger {
     const ERROR: number;
     const FATAL: number;
 
+    const levelFromName: { [name: string]: number };
```

Comment:
Instead of `name: string`, it should probably be a union type of the various level strings. So you could probably just split up `LogLevel` into `LogLevelString`, use `LogLevelString` here and then define `LogLevel` as `LogLevelString | number`